ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/GoogleIntegrationForm.tsx

```diff
@@ -148,6 +151,24 @@ function GoogleIntegrationForm({
     return (
         <>
             <PageSection variant="light" isFilled hasOverflowScroll>
+                <Alert
+                    title="Deprecation notice"
+                    component="p"
+                    variant={AlertVariant.warning}
+                    isInline
+                    className="pf-v5-u-mb-lg"
+                >
+                    <Text>Google Container Registry will be removed in a future release.</Text>
+                    <Text>
+                        It is recommended to use Google Artifact Registry as a registry replacement
+                        and Scanner V4 as a scanner replacement.
+                    </Text>
+                    <Text>
+                        See
+                        https://cloud.google.com/container-registry/docs/deprecations/container-registry-deprecation
+                        for more information.
```

Comment:
For your info, not a change request: pattern for both text and markup of external links is available on request.

Reply:
I plan on editing the message based on PM feedback, so I might as well do this change, too. Can you let me know how to represent this better?

Reply:
Thank you for asking, sent links via DM.